adaptors/library/jobs/satusehat-create-condition.js

```diff
@@ -0,0 +1,97 @@
+// Create a diagnosis in satusehat
+
+fn(state => {
+  state.conditionMappedData = state.visit.diagnosis.flatMap(diagnosis =>
+    state.lookup_table.dentist_diagnosis
+      .filter(item => diagnosis === item.fields.name)
```

Comment:
Can we use a better name than `item` here? The item is a diagnosis in the lookup table right?